### PR TITLE
Add dmtcp_command option to print list of connected workers

### PR DESCRIPTION
**Pull request by rohgarg**
This fixes issue #356.

### CODE REVIEW

File: src/coordinatorapi.cpp [hunk 1]
```diff
@@ -226,16 +226,17 @@ void CoordinatorAPI::updateSockFd()
   JASSERT(_coordinatorSocket == PROTECTED_COORD_FD);
 }
 
-void CoordinatorAPI::connectAndSendUserCommand(char c,
+char* CoordinatorAPI::connectAndSendUserCommand(char c,
```
Comment (karya0) [inline src/coordinatorapi.cpp]: Let's change the return type to `string` if possible.

File: src/coordinatorapi.cpp [hunk 2]
```diff
@@ -257,11 +258,11 @@ void CoordinatorAPI::connectAndSendUserCommand(char c,
   //the coordinator will violently close our socket...
   if (c=='q' || c=='Q') {
     *coordCmdStatus = CoordCmdStatus::NOERROR;
-    return;
+    return replyData;
   }
 
   //receive REPLY
-  recvMsgFromCoordinator(&reply);
+  recvMsgFromCoordinator(&reply, (void**)&replyData);
```
Comment (karya0) [inline src/coordinatorapi.cpp]: Need to call free on the buffer.

File: src/coordinatorapi.cpp (hunk 1 above)
Comment (rohgarg) [inline src/coordinatorapi.cpp]: Not doing it for now.

File: src/coordinatorapi.cpp (hunk 2 above)
Comment (rohgarg) [inline src/coordinatorapi.cpp]: Done in the new version.

**Pull request merged**